content/en/docs/setup/production-environment/turnkey/tencent.md

```diff
@@ -0,0 +1,18 @@
+## Running Kubernetes on Tencent Kubernetes Engine
+
+### Tencent Kubernetes Engine
+
+ [Tencent Cloud Tencent Kubernetes Engine (TKE)](https://intl.cloud.tencent.com/product/tke) provides native Kubernetes container management services. You can deploy and manage a Kubernetes cluster with TKE in just a few steps. For detailed directions, see [Deploy Tencent Kubernetes Engine](https://intl.cloud.tencent.com/document/product/457/11741).
+
+ In addition to full compatibility with the native Kubernetes API,TKE features Kubernetes plugins of Tencent Cloud such as [Cloud Block Storage](https://intl.cloud.tencent.com/product/cbs) and [Cloud Load Balancer](https://intl.cloud.tencent.com/product/clb), helping you build a complete system of container services and functions.
```

Comment:
nit:
```suggestion
 In addition to full compatibility with the native Kubernetes API, TKE features Kubernetes plugins for Tencent Cloud such as [Cloud Block Storage](https://intl.cloud.tencent.com/product/cbs) and [Cloud Load Balancer](https://intl.cloud.tencent.com/product/clb), helping you build a complete system of container services and functions.
```

If TKE is a certified solution, mention that and simplify the text around API compatibility. At the moment this reads too much like marketing text.

Reply:
@sftim Modified according to your suggestions. Please review.

Reply:
@sftim  What is your opinion on these changes? Is it suitable?
![image](https://user-images.githubusercontent.com/50287034/61378535-2b0a2c00-a8d8-11e9-82c0-6ea82992ef6c.png)